.coafile

```diff
@@ -1,7 +1,7 @@
 [DOCS]
 bears = SpaceConsistencyBear, LineLengthBear
 
-files = **/*.rst, README.md
+files = **/*.rst
```

Comment:
the problem is, thatin python and shell globs (I don't know other implementations) apparently '**' only matches files and dirs/subdirs, but NOT files in subdirs

so in shell:
`
fabian ~/devel/coala ⎇master  > echo **.rst   
CONTRIBUTING.rst README.rst RELEASE_NOTES.rst
fabian ~/devel/coala ⎇master  > echo **/*.rst
CONTRIBUTING.rst docs/index.rst README.rst RELEASE_NOTES.rst
`

we are implementing a slightly different behaviour with a bug, where '**' matches 'everything' (but apparently excluding the current dir) and the '/*.rst' then only looks for rst files in the subdirs of the current dir.